src/util/treeJsonProcessing.js

```diff
@@ -16,7 +16,7 @@ const pseudoRandomName = () => (Math.random()*1e32).toString(36).slice(0, 6);
  *
  * side-effects: node.hasChildren (bool) and node.arrayIdx (INT) for each node in nodes
  */
-const processNodes = (nodes) => {
+export const processNodes = (nodes) => {
```

Comment:
When I extended this to also return `nodeAttrKeys` I didn't foresee it being used outside of the initial dataset load. Rather than calling this function, why not simply add the incoming `action.newNodeAttrs` to the existing ones?

Reply:
Ah, I honestly just didn't dig deep enough here! Updated in [force-push](https://github.com/nextstrain/auspice/compare/0f2a10fcd11b501f696279d63996c4d871c57a3d..c2125254e251ed1cda169d88128280389f46dccf)